src/js/components/helptips/onboardingcompletetip.js

```diff
@@ -75,6 +75,7 @@ export class OnboardingCompleteTip extends React.Component {
             might have to refresh the page.
           </p>
           <p>You&apos;ve now got a good foundation in how to use Mender. Look for more help hints in the UI as you go along.</p>
+          <p>If you used one of our pre-built images you can start using full-image and delta updates right away.</p>
```

Comment:
The delta updates is an Enterprise feature, is this message only shown on Hosted Mender?

Reply:
👌 made it conditional and it will only be visible on hosted now - although it might also be an upselling point if it would show up in OS as well